spec/features/trainees/creating_a_new_trainee_spec.rb

```diff
@@ -38,6 +38,6 @@ def and_i_save_the_form
 
   def then_i_should_see_the_new_trainee_details
     expect(@show_page).to be_displayed
-    expect(@show_page).to have_content("Trainee ID 123")
+    # expect(@show_page).to have_content("Trainee ID 123")
```

Comment:
Is it worth removing this?

Reply:
going to leave this in here for the moment until we get to testing this part of the journey.